webhook_to_fedora_messaging/endpoints/message.py

```diff
@@ -1,43 +1,44 @@
+from fastapi import APIRouter, Depends, HTTPException, Request
 from fedora_messaging import api
-from flask import abort, Blueprint
-from sqlalchemy import select
-from sqlalchemy.exc import NoResultFound
+from starlette.status import HTTP_202_ACCEPTED, HTTP_400_BAD_REQUEST
 
+from webhook_to_fedora_messaging.endpoints.models.message import MessageRequest, MessageResult
+from webhook_to_fedora_messaging.endpoints.util import return_service_from_uuid
 from webhook_to_fedora_messaging.exceptions import SignatureMatchError
+from webhook_to_fedora_messaging.models import Service
 
-from ..database import db
-from ..models.service import Service
-from .parser.parser import msg_parser
-from .util import validate_request
+from .parser import parser
 
 
-message_endpoint = Blueprint("message_endpoint", __name__)
+router = APIRouter(prefix="/messages")
 
 
-@message_endpoint.route("/<service_uuid>", methods=["POST"])
-@validate_request([])
-def create_msg(service_uuid):
+@router.post(
+    "/{uuid}",
+    status_code=HTTP_202_ACCEPTED,
+    response_model=MessageResult,
+    tags=["messages"],
+)
+async def create_message(
+    body: MessageRequest,
```

Comment:
The `MessageRequest` validator object will enforce a format like:
```py
{
    data: {
        ...anything
    }
}
```

We can't expect external webhooks to adhere to that. I think we should replace the `MessageRequest` object here by `dict`, and actually remove the `MessageRequest` object entirely because it'll be the same problem in other functions: we don't control the content of the webhook.